### PR TITLE
Build install page

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
- Created the /install page
- Structured the site to have a common navigation
- Added the OS detection to the tabed install instructions

## QA
- Go to /install
- Check it matches [the copydoc](https://docs.google.com/document/d/1u8j5uN5d8qSEPDAjA03CGLeTJGf2p2MnHaKARkVmy7w/edit#heading=h.nel9e5tlwcfr)
- Check it [matches the design](https://camo.zenhubusercontent.com/79d4720049bfd269fc3b0beaa346535489f8f839/68747470733a2f2f757365722d696d616765732e67697468756275736572636f6e74656e742e636f6d2f393130353939392f35363336343733302d33343062333438302d363165372d313165392d393663372d6232363337653165643235392e706e67) _Agreed to change the instructions section to grey with hover and active state cards_
- Click on the different OS installs and check it jumps to the instructions about them
- Check loading the page with a hash auto selects the OS
- If you can check on a different OS for the auto detection (or just check the `os` variable in the page scripts)
- Check small screens work

Fixes https://github.com/canonical-web-and-design/multipass.run/issues/14

### CODE REVIEW

**Comment by anthonydillon** [comment]
@spencerbygraves I've updated the linux image and fixed the mac detection.

**Comment by spencerbygraves** [comment]
@anthonydillon small screen still has the old linux image. Thanks.

**Comment by anthonydillon** [comment]
@spencerbygraves good spot, done

File: index.html [hunk 1]
```diff
@@ -1,142 +1,125 @@
...
-  <section class="p-strip">
-    <div class="row">
-      <div class="col-12">
-        <h2 style="font-weight: 100;">Get started</h2>
-        <p>Multipass is available for <a href="/install#macOS">macOS</a> and <a href="/install#windows">Windows</a>, and the many <a href="/install#snap">Linux distributions</a> that support <a href="https://snapcraft.io/" class="p-link--external">snaps</a>.</p>
-      </div>
+<section class="p-strip">
+  <div class="row">
+    <div class="col-12">
+      <h2 style="font-weight: 100;">Get started</h2>
```
Comment (b-m-f) [inline index.html]: why put this inline?

Comment (b-m-f) [reply index.html]: There are q few more of these in this and other files

File: install.html [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+    }
+
+    // Select the current tab to selected
+    selectedTab.setAttribute('aria-selected', 'true');
+
+    // Reveal the assosiated tabpanel
+    selectedPanel.setAttribute('aria-hidden', 'false');
+  }
+
+  const hash = window.location.hash;
```
Comment (b-m-f) [inline install.html]: Using const here. Shouldnt we only use ES5 for IE support?

File: index.html (hunk 1 above)
Comment (anthonydillon) [reply index.html]: This is a quick hack used in other places until Vanilla 2.0 is released.

**Comment by b-m-f** [comment]
:+1:  LGTM. QAed, small screen works, but no selection for OS + jump is present.

Instead all 3 options are stacked and visible on the page.

**Pull request merged**